consensus/e2e_tests/utils_test.go

```diff
@@ -673,68 +700,78 @@ func WaitForNodeToSync(
 	unsyncedNode *shared.Node,
 	allNodes IdToNodeMapping,
 	targetHeight uint64,
-) error {
+) {
 	currentHeight := unsyncedNode.GetBus().GetConsensusModule().CurrentHeight()
```

Comment:
Optional improvement: You can call `t.Helper()` to improve logging.

https://pkg.go.dev/testing#B.Helper

Ditto elsewhere but not a must.